app/controllers/api/applications_controller.rb

```diff
@@ -51,9 +51,12 @@ def edit; end
 
   private
 
+  def accessible_provided_cinstances
```

Comment:
Isn't the name _provided_ too attached to the domain? Anyone not knowing what is a "provided" cinstance will be confused. The title could reflect what the method does: _get cinstances where account is not master_.

Reply:
Conclusion after speaking face to face: https://github.com/3scale/porta/pull/307#discussion_r232974829